src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1154,6 +1215,15 @@ private void InitEditor()
             gui.UpdatePlayerPatch(targetPatch);
         }
 
+        // Setup the display cell
+        previewMicrobe = (Microbe)microbeScene.Instance();
+        previewMicrobe.IsForPreviewOnly = true;
+        AddChild(previewMicrobe);
```

Comment:
I think this should be added underneath the editor world node (the same node where the preview hexes are added).